pkg/gofr/gofr.go

```diff
@@ -244,6 +254,17 @@ func (a *App) Shutdown(ctx context.Context) error {
 	return err
 }
 
+func isPortAvailable(port int) bool {
+	conn, err := net.DialTimeout("tcp", fmt.Sprintf(":%d", port), checkPortTimeout)
+	if err != nil {
+		return true
+	}
+
+	conn.Close()
+
+	return false
+}
```

Comment:
Reopening as previous was resolved without changes

https://github.com/gofr-dev/gofr/pull/1319#discussion_r1891210360

Please move this and the relative test to internal/port.go and port_test.go

Reply:
I did make it a function. And I will not be moving it to internal....as it does not exist as of now ....testutils exist... that is more related to helper functions for test. So adding it there does not make sense.

Reply:
I might have misunderstood something then. I'm sorry